modules/ROOT/pages/apply-tokenization-policy-to-api.adoc

```diff
@@ -0,0 +1,27 @@
+= Use Tokenization and Detokenization as API Gateway Policies
+
+In this scenario, you apply the tokenization policy to the API gateway, which serves as the intermediary between the Edge and the Mule application. The tokenization service in this case is implicit and made available through internal integration with the API gateway. 
+
+== Prerequisites
+
+To apply a tokenization policy to the API Gateway, you must meet these requirements:
+
+* You must have the following Anypoint permissions:
+ ** Manage Policies
+* The API must be managed in Mule 4 or later
+
+== Apply a Tokenization Policy to an API Gateway
+
+When the tokenization service is applied directly to the API gateway, you don't need to use the tokenization RAML to expose the tokenization service. You only need to set up the API gateway for the Mule app to serve as the proxy.
+
+These are the high level steps:
```

Comment:
Self-referential. How about To apply (a? the?) tokenization service to an API gateway:  or similar.